Tests/conf.json

```diff
@@ -3601,7 +3600,8 @@
         "GoogleKubernetesEngine": "SDK",
         "TAXIIFeed": "Cannot use proxy",
         "EWSO365": "oproxy dependent",
-        "MISP V2": "Cleanup process isn't performed as expected."
+        "MISP V2": "Cleanup process isn't performed as expected.",
+        "GitHub": "Cannot use proxy"
```

Comment:
because the command `GitHub-get-stale-prs` failed with this error: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /search/issues?q=repo%3Acontent-bot%2Fhello-world+is%3Aopen+updated%3A%3C2021-05-25T12%3A42%3A05+is%3Apr&per_page=100 (Caused by ProxyError('Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')))

Reply:
was the error consistent?

Reply:
yes, you can see it in the [build](https://app.circleci.com/pipelines/github/demisto/content/86075/workflows/90d342e3-d1f1-421f-90ec-2db0926fbb7d)